### PR TITLE
allow Apollo to onboard with any stake

### PR DESCRIPTION
**Pull request by Anze1m**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] index.js exports updated
- [x] Automatic tests added/updated
- [x] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [x] Version updated

### CODE REVIEW

File: test/actions/onboard_actions.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ describe('Onboard Actions', () => {
     });
 
     it('fails if the stakeAmount is not matching the whitlisted pledge', async () => {
-      kycWhitelistWrapperMock.getRequiredDeposit.resolves('45678');
+      kycWhitelistWrapperMock.getRequiredDeposit.resolves('456780');
```
Comment (krzysztof-jelski) [inline test/actions/onboard_actions.js]: Why change the number here? I think it's unnecessary.

File: test/contracts/front/roles.js [hunk 2]
```diff
@@ -249,6 +249,13 @@ describe('Roles Contract', () => {
         expect(await getRole(apollo)).to.equal(APOLLO.toString());
       });
 
+      it('allows to onboard with bigger stake', async () => {
```
Comment (krzysztof-jelski) [inline test/contracts/front/roles.js]: Perhaps move this test after `adds node to validator set and block rewards`. It kind of builds upon it and `stores the role and deposit` - it contains assertions for both checking if onboarding is successful and for beneficiaryShare.
